backend/shared/src/resolve-market-helpers.ts

```diff
@@ -502,6 +504,20 @@ export const payUsersTransactions = async (
     }
   }
 
+  if (payoutCash && cashoutable) {
+    const limits = await pg.manyOrNone(
+      `select id, (data->'sweepstakes5kLimit')::boolean as is_limited from users where id = any($1)`,
+      [mergedPayouts.map((p) => p.userId)]
+    )
+    const limitsById = Object.fromEntries(
+      limits.map((l) => [l.id, l.is_limited])
+    )
+    txns.forEach((t) => {
+      t.data!.cashoutable = true
```

Comment:
This approach also makes it necessary that we have a `temporarySweepstakesStatus` in case they have a sus session and they're marked as not `sweepstakesVerified` temporarily